edx_notifications/server/web/static/edx_notifications/js/views/counter_icon_view.js

```diff
@@ -0,0 +1,66 @@
+var CounterIconView = Backbone.View.extend({
```

Comment:
Yes, we have a story in this sprint to add the Jasmine (and BokChoy) test harness in edx-notifications. Right now we are interested in generic high-level feedback on the Backbone patterns we are setting up.